reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -49,9 +49,13 @@
  * reactor.core.publisher.Flux#publishOn(Scheduler) publishOn} or {@link reactor.core.publisher.Mono#subscribeOn
  * subscribeOn} :
  * <p>
- * <ul> <li>{@link #fromExecutorService(ExecutorService)}}. </li> <li>{@link #newParallel}
- * : Optimized for fast {@link Runnable} executions </li> <li>{@link #single} : Optimized
- * for low-latency {@link Runnable} executions </li> <li>{@link #immediate}. </li> </ul>
+ * <ul>
```

Comment:
since we're touching this one, perhaps we can add the boundedElastic here?

Reply:
boundedElastic was introduced in 3.3, this is targeting 3.2

Reply:
Was confused by:

> simonbasle added this to the 3.3.2.RELEASE milestone 41 minutes ago